### PR TITLE
Adding module for creating test reports in ReportPortal

### PR DESCRIPTION
**Pull request by esu-justinas**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -57,7 +57,7 @@
 *.bat text
 *.ep text
 *.css text
-*.config text
+*.configuration text
```
Comment (ricardas-buc) [inline .gitattributes]: Is this is intentional change?

File: test-automation-reportportal/src/main/kotlin/com/nortal/test/report/reportportal/CustomScenarioReporter.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package com.nortal.test.report.reportportal
+
+import com.epam.reportportal.cucumber.ScenarioReporter
+import com.epam.reportportal.listeners.ListenerParameters
+import com.epam.reportportal.service.ReportPortal
+import com.epam.reportportal.utils.properties.PropertiesLoader
+
+class CustomScenarioReporter : ScenarioReporter() {
```
Comment (ricardas-buc) [inline test-automation-reportportal/src/main/kotlin/com/nortal/test/report/reportportal/CustomScenarioReporter.kt]: custom does not mean much, should have a better name.

File: settings.gradle.kts [hunk 3]
```diff
@@ -89,6 +91,9 @@ dependencyResolutionManagement {
             library("allure-plugin-api", "io.qameta.allure", "allure-plugin-api").versionRef("allure")
             library("allure-commandline", "io.qameta.allure", "allure-commandline").versionRef("allure")
 
+            library("reportportal-cucumber", "com.epam.reportportal", "agent-java-cucumber6").versionRef("reportportal")
```
Comment (ricardas-buc) [inline settings.gradle.kts:94]: I have seen that both junit5 and allure modules are available. Have you checked what they provide compared to vanilla java lib? see: https://github.com/reportportal/allure-java

**Review by ricardas-buc — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (esu-justinas) [reply .gitattributes]: Seems like this was unintentional.

File: settings.gradle.kts (hunk 3 above)
Comment (esu-justinas) [reply settings.gradle.kts:94]: Took a look at this module. It seems that it is still in BETA version, without any relevant documentation on how to use it. But I'll try to look into it a little deeper, maybe I'll manage to figure out how to integrate it with our allure reports.

**Comment by esu-justinas** [comment]
Did some investigation on https://github.com/reportportal/allure-java , but it is pretty difficult navigating it without any documentation. It seems that junit5 module in that repo enables allure annotations, which we don't use anyhow at the moment. As for allure-cucumber-6 module, it is still unclear for me how to use it, or what purpose it plays.

So for now, leaving the vanilla java agent lib of ReportPortal.

**Review by ricardas-buc — APPROVED** [review]

**Pull request merged**